src/components/EventSearchInput/EventSearchInput.js

```diff
@@ -12,6 +12,21 @@ class EventSearchInput extends Component {
 
     return (
       <div className={styles.inputSearchWrapper}>
+        <svg
```

Comment:
Excellent suggestion. I found that there is a FaSearch which should work quite well. Latest adds it in. Attached is screenshot
<img width="832" alt="screen shot 2017-03-30 at 3 26 21 pm" src="https://cloud.githubusercontent.com/assets/5664775/24522446/94e2a94c-155d-11e7-93b3-2e8f5cd98a6f.png">